build_web_compilers/lib/src/common.dart

```diff
@@ -18,6 +18,12 @@ final sdkDir = p.dirname(p.dirname(Platform.resolvedExecutable));
 String defaultAnalysisOptionsArg(ScratchSpace scratchSpace) =>
     '--options=${scratchSpace.fileFor(defaultAnalysisOptionsId).path}';
 
+String sdkDdcKernelPath(bool soundNullSafety) => p.url.join('lib', '_internal',
+    soundNullSafety ? 'ddc_outline_sound.dill' : 'ddc_sdk.dill');
+
+String soundnessExt(bool soundNullSafety) =>
+    soundNullSafety ? '.sound' : '.unsound';
```

Comment:
This choice doesn't bother me, I just wanted to point out that everywhere else this is referred to as "weak" instead of "unsound".

Reply:
hmm, `unsound` seems to make a lot more sense lol. cc @natebosch wdyt?

Reply:
I prefer "sound" and "unsound", if we come across places still using "strong" and "weak" we should change them.